control-plane/agents/src/bin/core/controller/policies/rebuild_policies.rs

```diff
@@ -16,7 +17,9 @@ use std::time::Duration;
 */
 
 // time constants, in seconds
+/// A time period optimized for rebuild performance i.e. preferring log-based rebuild.
```

Comment:
Btw since you're modifying this file, the comment above with c style comments, can you move it to Line1 as a doc-comment: `//!` ?

Reply:
Not showing up as moved, though it may be the github ui not updating

Reply:
It wasn't the github ui but me :) I had moved it but missed to change the comment type. Corrected now.